src/main/java/org/launchcode/techjobs/oo/test/JobTest.java

```diff
@@ -1,12 +1,155 @@
 package org.launchcode.techjobs.oo.test;
 
+import org.junit.Assert;
+import org.junit.Test;
+//import org.junit.Before;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.launchcode.techjobs.oo.*;
+
+import static org.junit.Assert.*;
 
 /**
  * Created by LaunchCode
  */
 @RunWith(JUnit4.class)
+
 public class JobTest {
+    Job job1;
+    Job job2;
+    Job test_job1;
+    Job test_job2;
+    Job test_job3;
+    Job test_job4;
+    Job test_job5;
+
+    /*@Before
+        public void createJobObject(){
+            job1 = new Job();
+            job2 = new Job();
+            test_job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+            test_job2 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+            test_job3 =new Job("", new Employer(""), new Location(""), new PositionType(""), new CoreCompetency(""));
+    }
+
+     */
+
+    @Test
+    public void testJobConstructorSetsAllFields(){
+        test_job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence") );
+
+        assertTrue(test_job1.getId() > 0);
+        assertEquals("Product tester", test_job1.getName());
+        assertEquals("ACME", test_job1.getEmployer().getValue());
+        assertTrue(test_job1.getEmployer() instanceof Employer);
+        assertEquals("Desert", test_job1.getLocation().getValue());
+        assertTrue(test_job1.getLocation() instanceof Location);
+        assertEquals("Quality control", test_job1.getPositionType().getValue());
+        assertTrue(test_job1.getPositionType() instanceof PositionType);
+        assertEquals("Persistence", test_job1.getCoreCompetency().getValue());
+        assertTrue(test_job1.getCoreCompetency() instanceof CoreCompetency);
+
+
+    }
+/*
+    @Test
+    public void testPositionTypeEqualsMethod() {
+        Job test_job4 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence") );
+        assertTrue(test_job4.getPositionType() instanceof PositionType);
+
+        assertEquals("Quality control", test_job4.getPositionType().getValue());
+    }
+
+ */
+
+    @Test
+    public void testSettingJobId(){
+        Job job1 = new Job();
```

Comment:
By doing `Job job1 = ...`, you're declaring the new variable. If you had done instead `job1 = new Job();`, then it would be using the global variables referenced at the beginning of the class. Just more of an FYI :)